Antidote/ActiveCallViewController.m

```diff
@@ -89,22 +94,23 @@ - (void)createContainerView
 
 - (void)createMicrophoneButton
 {
-    self.microphoneButton = [self createButtonWithImageName:@"call-microphone-enable" action:@selector(toggleMicrophone)];
+    self.microphoneButton = [self createButtonWithImageName:@"call-microphone-disable" action:@selector(micButtonPressed)];
 
-    UIImage *selectedImage = [UIImage imageNamed:@"call-microphone-disable"];
+    UIImage *selectedImage = [UIImage imageNamed:@"call-microphone-enable"];
     [self.microphoneButton setImage:selectedImage forState:UIControlStateSelected];
+    self.microphoneButton.selected = YES;
 
     [self.containerView addSubview:self.microphoneButton];
 }
 
 - (void)createMuteButton
 {
-    self.muteButton = [self createButtonWithImageName:@"call-audio-enable" action:@selector(toggleMute)];
+    self.speakerButton = [self createButtonWithImageName:@"call-audio-enable" action:@selector(speakerButtonPressed)];
 
     UIImage *selectedImage = [UIImage imageNamed:@"call-audio-disable"];
-    [self.muteButton setImage:selectedImage forState:UIControlStateSelected];
+    [self.speakerButton setImage:selectedImage forState:UIControlStateSelected];
```

Comment:
microphoneButton uses `enable` image for selected state, speakerButton uses `disable`. Maybe it would be better to use same approach in both cases?